### DISCUSSION TITLE
[Tour Segmenter] Elevation Gain between L and R in Segmenter

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#975] [General]
I do not understand how the Elevation Gain is computed in the Segmenter.  Even though there is gain from Marker to Marker defined by L and R borders, it is not reflected properly in the chart.  See example below which includes a snip-it of the total elevation gain from the Tour Book view. 
* Should the summary line of the Segmenter be able to show total gain?
* If I select or move from Segment to Segment in the Segmenter, the L & R markers are correctly repositioned on the (2-D) Map and Tour Chart, but I don't understand the computation of the Elevation gain for the segment.

![Elevation Gain](https://user-images.githubusercontent.com/106929478/207428312-4e7466c7-55ce-4bb4-8232-71cee783d6b9.jpg)

**Comment by wolfgang-ch** [comment]
> I do not understand how the Elevation Gain is computed in the Segmenter

Have you read the whole documentation https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tour-segmenter 

You can show each segment in the tour chart or tour editor and can identify each meter/feet. This is the reason why I implemented it that the tour segmenter is not a black box.

>Should the summary line of the Segmenter be able to show total gain?

Currently the totals are not yet computed/displayed for all columns in the tour segmenter

>If I select or move from Segment to Segment in the Segmenter, the L & R markers are correctly repositioned on the (2-D) Map and Tour Chart, but I don't understand the computation of the Elevation gain for the segment.

Elevation gain is computes with this algorithm

![image](https://user-images.githubusercontent.com/1283445/207521336-6112a42a-9641-4d5a-b4f9-70be696b96a1.png)
